pkg/nodeset/v1alpha1/types.go

```diff
@@ -181,6 +186,9 @@ type NodeSetStatus struct {
 	Conditions []NodeSetCondition `json:"conditions,omitempty"`
 }
 
+// +genclient=true
```

Comment:
Right. NodeSet shouldn't be namespaced if Nodes aren't. However, in Archon, we support namespaced Instances and InstanceGroups. So different clusters can naturally exist in their own namespaces. FWIW.